fitbenchmarking/fitting/mantid/tests/test_prepare_data.py

```diff
@@ -17,11 +17,27 @@
 from fitting.mantid.prepare_data import setup_errors
 from fitting.mantid.prepare_data import convert_back
 
-from utils import fitbm_problem
+from parsing.fitbm_problem import BaseFittingProblem
 
 
 class MantidTests(unittest.TestCase):
 
+    def misra1a_file(self):
```

Comment:
Could misrela_file, neutron_peak_19_file and similar be factored out into separate python file. If only used within mantid/tests I suggest put these in python file named data_for_testing.py